=== modified file 'bin/nova-manage'
Index: nova-2011.3~bzr1054/bin/nova-manage
===================================================================
--- nova-2011.3~bzr1054.orig/bin/nova-manage
+++ nova-2011.3~bzr1054/bin/nova-manage
@@ -1098,7 +1098,7 @@
 
 def main():
     """Parse options and call the appropriate class/method."""
-    utils.default_flagfile()
+    utils.default_flagfile('/etc/nova/nova.conf')
     argv = FLAGS(sys.argv)
     logging.setup()
 
